src/main.c

```diff
@@ -775,10 +773,10 @@ void setup_main_app (GromitData *data, int argc, char ** argv)
   data->modified = 0;
 
   data->default_pen =
-    paint_context_new (data, GROMIT_PEN, data->red, 7,
+    paint_context_new (data, GROMIT_PEN, gdk_rgba_copy(data->red), 7,
```

Comment:
manpage says "The result must be freed through gdk_rgba_free()." Should be added to paint-context_free() then.